test/service/test_drs.py

```diff
@@ -140,46 +141,63 @@ def test_drs(self):
                         # We expect a DRS object with an access URL
                         expected['access_methods'] = [
                             {
-                                'access_url': {
-                                    'url': ('https://org-hca-dss-checkout-prod.s3.amazonaws.com/'
-                                            'blobs/307.a72.eb6?foo=bar&et=cetera')
-                                },
+                                'access_url': {'url': 'https://org-hca-dss-checkout-prod.s3.amazonaws.com/'
+                                                      'blobs/307.a72.eb6?foo=bar&et=cetera'},
                                 'type': 'https'
+                            },
+                            {
+                                'access_url': {'url': 'gs://important-bucket/object/path'},
+                                'type': 'gs'
                             }
                         ]
                         self.assertEqual(drs_object, expected)
                         # self.assertEqual(sort_frozen(freeze(drs_object)), sort_frozen(freeze(expected)))
                     else:
-                        # DRS object should have an access ID
+                        # The access IDs are so similar because the mock tokens are the same
```

Comment:
Might want to point out the difference as in

```suggestion
                        # The access IDs are so similar because the mock tokens are the same …
```